conftest.py

```diff
@@ -617,18 +616,11 @@ def _update_os_related_config():
         log_file=tests_log_file,
         log_level=session.config.getoption("log_cli_level") or logging.INFO,
     )
-    py_config_scs = py_config.get("storage_class_matrix", [])
-
-    # Set py_config["storage_class_matrix"]
-    if not skip_if_pytest_flags_exists(pytest_config=session.config):
-        cluster_storage_classes_names = [sc.name for sc in list(StorageClass.get(dyn_client=get_client()))]
-        # If TOPOLVM storage class is present in the cluster - add it to the matrix
-        # And do not add the HPP storage classes
-        if StorageClassNames.TOPOLVM in cluster_storage_classes_names:
-            py_config_scs.extend(py_config["topolvm_storage_class_matrix"])
-        # If HPP CSI storage class is present - add HPP CSI storage classes
-        elif HppCsiStorageClass.Name.HOSTPATH_CSI_BASIC in cluster_storage_classes_names:
-            py_config_scs.extend(py_config["new_hpp_storage_class_matrix"])
+    # Add HPP-CSI-BASIC/HPP-CSI-PVC-BLOCK to global config's storage_class_matrix, only
+    # if command line option --storage-class-matrix includes them:
+    py_config_scs = update_storage_class_matrix_config(
+        session=session, pytest_config_matrix=py_config.get("storage_class_matrix", [])
+    )
```

Comment:
`$ uv run pytest -k test_empty_url --tc-file=tests/global_config.py --collect-only` - picks only OCS, and doesn't check for HPP presence.. means there may or may not be HPP pods. Is that okay for IUO tests?

This also picks only OCS
`$ uv run pytest -k test_empty_url --tc-file=tests/global_config.py --storage-class-matrix=ocs-storagecluster-ceph-rbd-virtualization --collect-only`
So also, HPP pods may or may not be there. Is that okay?

Reply:
This is the correct behavior. We should not go by what is in the cluster. if someone wants to run tests against hpp, they must either use a config with HPP or pass HPP via command line. ```uv run pytest -k test_empty_url --tc-file=tests/global_config.py --storage-class-matrix=ocs-storagecluster-ceph-rbd-virtualization,hostpath-csi-basic,hostpath-csi-pvc-block -svv --collect-only```

Reply:
Right, but the main goal of this PR, as I understood, was to fix the IUO tests that expect HPP pods by default. But still, if someone will run with just OCS storage class - HPP pods may not be there

Reply:
But at least it's fixed for SNO now

Reply:
yeah. IUO team would need to change the matrix for global_config.py